compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/CodeAnalyzer.java

```diff
@@ -3414,7 +3409,8 @@ public void visit(BLangAnnotAccessExpr annotAccessExpr, AnalyzerData data) {
     @Override
     public void visit(BLangRegExpTemplateLiteral regExpTemplateLiteral, AnalyzerData data) {
         List<BLangExpression> interpolationsList =
-                symResolver.getListOfInterpolations(regExpTemplateLiteral.reDisjunction.sequenceList);
+                symResolver.getListOfInterpolations(regExpTemplateLiteral.reDisjunction.sequenceList,
```

Comment:
Check this also if this is related to IDL client, IMO it is not.

Reply:
This is related to reg exp.